spec/classes/mysql_server_backup_spec.rb

```diff
@@ -43,13 +43,26 @@
           )}
 
           it 'should have compression by default' do
-            is_expected.to contain_file('mysqlbackup.sh').with(
-              :content => /bzcat -zc/
+            is_expected.to contain_file('mysqlbackup.sh').with_content(
+              /bzcat -zc/
```

Comment:
I notice that some of the other tests were moving from ```with``` to ```with_content```.  For example the ```should be able to backup events table``` test on line 123 was already using ```with_content```, but ```should skip backing up events table by default``` on line 51 wasn't, so I changed it to be consistent with other tests.  

I hope that wasn't overstepping...